app/feature/feature-forever/src/main/kotlin/com/hedvig/android/feature/forever/di/ForeverModule.kt

```diff
@@ -1,22 +1,20 @@
 package com.hedvig.android.feature.forever.di
 
 import com.hedvig.android.apollo.giraffe.di.giraffeClient
-import com.hedvig.android.feature.forever.GetReferralsInformationUseCase
-import com.hedvig.android.feature.forever.ReferralsViewModel
-import com.hedvig.android.feature.forever.data.ReferralsRepository
-import com.hedvig.android.language.LanguageService
+import com.hedvig.android.feature.forever.ForeverViewModel
+import com.hedvig.android.feature.forever.data.GetReferralsInformationUseCase
 import org.koin.androidx.viewmodel.dsl.viewModel
 import org.koin.dsl.module
 
 val foreverModule = module {
```

Comment:
Could we check if the entire `foreverModule` can move inside the forever feature?
This is the thing that would allow everything else like the ViewModels and so on to be internal from that point on, and :app doesn't need to know about any of them.

Reply:
`ForeverRepository` is still used by payments and offer, is that okay or should we create separate use cases for those features?